pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1724,6 +1728,18 @@ type GatewaySpec struct {
 	AdditionalVolumeMounts AdditionalVolumeMounts `json:"additionalVolumeMounts,omitempty"`
 }
 
+// RGWLoggingSpec is intended to extend the s3/swift logging for client operations
+type OpsLogSidecar struct {
+
+	// Enable RGW Ops Logs for S3 in a sidecar named ops-log
+	// +optional
+	Enabled *bool `json:"enabled,omitempty"`
+
```

Comment:
Let's remove `enabled`. Presence of `opsLogSidecar` (non-`nil`) will indicate enabled. Non-presence (`nil`) will indicate non-presence.

```suggestion
```